### ISSUE TITLE
Missing quotes on save

### ISSUE THREAD

**Issue by underdpt** [absolutehh/dotenv-manipulator#9]
Hi,

When saving a .env, i noticed quotes around variables with spaces are missing. This leads to an 

`PHP Fatal error:  Uncaught ReflectionException: Class config does not exist in...`

I think the writer should quote variables with spaces, and maybe respect blank lines (to keep readability)

**Comment by Gummibeer** [comment]
I agree that strings should be quoted - but keeping variables isn't easy. 😕
